tests/components/airq/test_coordinator.py

```diff
@@ -0,0 +1,47 @@
+"""Test the air-Q data update coordinator."""
```

Comment:
You shouldn't be testing the coordinator as a standalone.
It should instead be part of a full test suite.

You would have a couple of tests in `test_init.py`:
- One for successful initialisation + unload (eg `test_setup_config_and_unload`)
- One for failed initialisation (eg `test_async_setup_entry_not_ready`)

And you would have a platform test `test_sensor.py`:
- Ensure that device_registry is correctly populated
- Ensure that entity_registry is correctly populated
- Ensure that state_machine is correctly populated

`sfr_box` tests were recently refactored to use `SnapshotAssertion` so I think it's a good base.

Anyway this is probably a big chunk for a follow-up PR, and I recommend that you remove this test file for now.

Reply:
Thanks for the advice, I'll look into it. Removed the file for now.

Reply:
You can add coordinator.py to coverage ignore file for now.

Reply:
Of course, thanks